modules/terraform/options.go

```diff
@@ -102,3 +106,144 @@ func WithDefaultRetryableErrors(t *testing.T, originalOptions *Options) *Options
 
 	return newOptions
 }
+
+// GetVariableAsStringFromVarFile Gets the string represention of a variable from a provided input file found in VarFile
+// For list or map, use GetVariableAsListFromVarFile or GetVariableAsMapFromVarFile, respectively.
+func GetVariableAsStringFromVarFile(t *testing.T, option *Options, fileName string, key string) string {
+	result, err := GetVariableAsStringFromVarFileE(t, option, fileName, key)
+	require.NoError(t, err)
+
+	return result
+}
+
+// GetVariableAsStringFromVarFileE Gets the string represention of a variable from a provided input file found in VarFile
+// Will return an error if GetAllVariablesFromVarFileE returns an error or the key provided does not exist in the file.
+// For list or map, use GetVariableAsListFromVarFile or GetVariableAsMapFromVarFile, respectively.
+func GetVariableAsStringFromVarFileE(t *testing.T, option *Options, fileName string, key string) (string, error) {
+	variables, err := GetAllVariablesFromVarFileE(t, option, fileName)
+
+	if err != nil {
+		return "", err
+	}
+
+	variable, exists := variables[key]
+
+	if !exists {
+		return "", InputFileKeyNotFound(key)
+	}
+
+	return fmt.Sprintf("%v", variable), nil
+}
+
+// GetVariableAsMapFromVarFile Gets the map represention of a variable from a provided input file found in VarFile
```

Comment:
NIT: in both descriptions, it's probably worth calling out that this is specifically a map of strings. For maps with more complicated nested types, or other types in general, use `GetAllVariablesFromVarFile`.